src/SpeechRecognition.js

```diff
@@ -108,6 +108,9 @@ export default function SpeechRecognition(options) {
 
       startListening = () => {
         if (recognition && !listening) {
+          if (!recognition.continuous) {
```

Comment:
Please update the `startListening` docs ([here](https://github.com/FoundersFactory/react-speech-recognition#startlistening-function)) to say that if the `continuous` option is set to false, it will reset the `transcript` prop.